src/pattern-library/components/patterns/input/InputPage.tsx

```diff
@@ -69,11 +69,44 @@ export default function InputPage() {
             presentational component props
           </Library.Link>
           .
+          <Library.Example title="feedback">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                Set <code>feedback</code> to indicate that there is an
+                associated error or warning for the <code>Input</code>.
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>
+                  {`"error"`} | {`"warning"`}
+                </code>
+              </Library.InfoItem>
+              <Library.InfoItem label="default">
+                <code>{`undefined`}</code>
+              </Library.InfoItem>
+            </Library.Info>
+
+            <Library.Demo withSource>
+              <div className="w-[350px]">
+                <Input
+                  aria-label="Input with error"
+                  feedback="error"
+                  value="something invalid"
+                />
+              </div>
+              <div className="w-[350px]">
+                <Input
+                  aria-label="Input with warning"
+                  feedback="warning"
+                  value="something invalid"
```

Comment:
```suggestion
                  value="might be a problem"
```

If the input was outright invalid, the "error" state should be used. The warning state is used when the input might cause a problem, but we're leaving it up to the user to decide to continue or change it.